src/ParodusInternal.h

```diff
@@ -31,6 +31,9 @@
 
 #include "wss_mgr.h"
 #include "parodus_log.h"
+#ifdef PARODUS_LOGGER
+#include <rdk_debug.h>
```

Comment:
I'm not thrilled about this way of using the rdk_debug functionality.  I'll work on a follow-up patch of how I think we should structure this code.